src/components/ToggleSwitch/index.tsx

```diff
@@ -214,7 +223,7 @@ const ToggleSwitch: React.FC<ToggleSwitchProps> = ({
           ]}
         />
       </Animated.View>
-      <Text style={[styles.labelRight, styleLabelRight]}>{labelRight}</Text>
+      {labelRight && <Text style={[styles.labelRight, styleLabelRight]}>{labelRight}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the conditional rendering of the `labelRight` text component based on its existence is also a good practice. This prevents unnecessary rendering and potential errors if `labelRight` is undefined or null.
```diff
+ {labelRight && <Text style={[styles.labelRight, styleLabelRight]}>{labelRight}</Text>}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->